### PR TITLE
adding an Istio day banner for 2024

### PR DESCRIPTION
**Pull request by nate-double-u**
## Description

This PR adds a banner for Istio Day EU, 2024-03-19.

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by nate-double-u** [comment]
I'm not sure why this failed.

```
./en/events/banners/istio-day-eu-2024.md:8: MD047 File should end with a single newline character
```

Added a newline at the end, but it looks like GH has stripped it out.

File: content/en/events/banners/istio-day-eu-2024.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Istio Day Europe
+period_start: 2024-02-21
+period_end: 2024-03-19
+link: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/
+---
+
+Join us for Istio Day Europe, a KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Event.
+Register now! 19 March 2024 | Paris, France
```
Comment (craigbox) [inline content/en/events/banners/istio-day-eu-2024.md:9]:
```suggestion
Register now! 19 March 2024 | Paris, France

```

**Comment by craigbox** [comment]
added newline

**Comment by nate-double-u** [comment]
We'll probably have to retest again as the update went in during it's run.

**Comment by dhawton** [comment]
> We'll probably have to retest again as the update went in during it's run.

The testing will see the new commit and trigger a test for it automatically.

File: content/en/events/banners/istio-day-eu-2024.md (hunk 1 above)
Comment (craigbox) [inline content/en/events/banners/istio-day-eu-2024.md:9]:
```suggestion
Join us for Istio Day Europe, a KubeCon + CloudNativeCon Europe Co-located Event. 19 March 2024, Paris, France. Register now! 
```

Comment (nate-double-u) [reply content/en/events/banners/istio-day-eu-2024.md]: Good suggestion, thx

File: content/en/events/banners/istio-day-eu-2024.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Istio Day Europe
+period_start: 2024-02-21
+period_end: 2024-03-19
+link: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/
+---
+
+Join us for Istio Day Europe, a KubeCon + CloudNativeCon Europe Co-located Event. 19 March 2024, Paris, France. Register now!
```
Comment (nate-double-u) [inline content/en/events/banners/istio-day-eu-2024.md:8]: Removing end of line whitespace.
```suggestion
Join us for Istio Day Europe, a KubeCon + CloudNativeCon Europe Co-located Event. 19 March 2024, Paris, France. Register now!
```

**Review by kfaseela — APPROVED** [review]

**Comment by nate-double-u** [comment]
Thanks everyone, I appreciate this!

**Pull request merged**